db/seeds.rb

```
@@ -1,58 +1,163 @@
+    end
+  end
+end
+
+def new_smartphone_released_by(brand)
+  os_name = 'Android'
+  # os_name = 'iOS' if brand.name == SmartphoneModel.APPLE
+  os_name = 'iOS' if brand.is_apple?
+
+  SmartphoneModel.create!(
+    name: model[:name],h
+    year_id: model[:year_id],
```

RuboCop (1.76.1)

[fatal] unexpected token tNL
(Using Ruby 3.2 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)